tests/migration/online_migration/register_without_ltss.pm

```diff
@@ -28,6 +28,15 @@ sub run {
     set_var('SCC_ADDONS', join(',', @scc_addons));
 
     register_system_in_textmode;
+
+    # Sometimes in HA scenario, we need to test rolling upgrade migration from 
+    # a LTSS version to another one LTSS version.
+    # In this case, we need to set back ltss to SCC_ADDONS and set HDDVERSION to
+    # the targeted OS version for getting the good LTSS regcode.
+    if (get_var('LTSS_TO_LTSS')) {
+        set_var('SCC_ADDONS', join(',', @scc_addons, 'ltss'));
+        set_var('HDDVERSION', get_var('UPGRADE_TARGET_VERSION', get_var('VERSION')));
```

Comment:
Needed for this [LTSS registration](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/registration.pm#L315-L319)